packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -804,16 +804,23 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
       const selectionEnd = element.selectionEnd;
       const isRangeSelected = selectionStart !== selectionEnd;
       const inputValue = element.value;
+      const shouldInputLoseFocusOnArrowKey = this.props.shouldInputLoseFocusOnArrowKey
+        ? this.props.shouldInputLoseFocusOnArrowKey(element)
+        : false;
```

Comment:
I don't love that the const shouldInputLoseFocusOnArrowKey is a boolean, but this.props.shouldInputLoseFocusOnArrowKey is a function returning a boolean. 

Can we either move this function down to 822, or come up with a different name stating that this is the result of the function?

Reply:
I have addressed this comment